SQF/dayz_code/configVariables.sqf

```diff
@@ -39,6 +39,10 @@ MaxAmmoBoxes = 3; // Max number of random Supply_Crate_DZE to spawn around the m
 MaxMineVeins = 50; // Max number of random mine veins to spawn around the map
 timezoneswitch = 0; // Changes murderMenu times with this offset in hours.
 DZE_SafeZonePosArray = []; //Prevent players in safeZones from being killed if their vehicle is destroyed. Format is [[[3D POS], RADIUS],[[3D POS], RADIUS]]; Ex. DZE_SafeZonePosArray = [[[6325.6772,7807.7412,0],150],[[4063.4226,11664.19,0],150]];
+DZE_SafeZoneNoBuildItems = []; // Array of object class names not allowed to build near the safe zones listed above. i.e ["VaultStorageLocked","LockboxStorageLocked","Plastic_Pole_EP1_DZ"] etc.
+DZE_SafeZoneNoBuildNear = []; //Array of object class names that are blacklisted to build near. i.e ["Land_Mil_ControlTower","Land_SS_hangar"] etc.
+DZE_SafeZoneNoBuildDistance = 1000; // Distance from safe zones listed above to disallow building near.
+DZE_BlacklistNoBuildDistance = 1000; // Distance from blacklisted objects to disallow building near.
```

Comment:
1000m is too high a default for this if it is only meant to prevent blocking off certain building types with bases. For a hangar something like 150m should work for that. For a barracks you would only need it to be about 25m. Otherwise it will be confusing to see the message when there is no building of that type in sight. It will unintentionally be blocking building around a lot of other building types in dense areas. 

Admins can always increase it if they want to prevent building a camp tower next to the barracks, etc.